http/1.6/modules/ROOT/pages/http-listener-configuration.adoc

```diff
@@ -0,0 +1,504 @@
+
+=== Create Sending HTTP Responses Mule App Example
+
+The following example shows how to configure HTTP responses for the HTTP *Listener* source in Studio. The example is a Mule app flow where an endpoint stores logos for an account:
+
+* When storage is successful, an HTTP regular response returns the status code `201`, the reason phrase `Created`, and the body `Corporate Logo has been stored as a MuleSoft logo`.
+* When storage fails, an HTTP error response returns a status code defined through the `errorCode` variable (if available) or `500` by default.
+** The custom header returns `X-Time`.
+** The body returns `Corporate Logo could not be stored`.
+** If there is a `CONNECTIVITY` error storing the logo, the returned status code is `504`, while any other errors result in a `500` status code.
+
+While a reason phrase is not defined for error responses, HTTP Connector attempts to define a reason phrase based on the status code. Thereby, a `Gateway Timeout` or `Internal Server Error` error can return for the previous explained HTTP responses scenarios.
+
+To create the Mule app in Studio, follow these steps:
+
+. In the *Mule Palette* view, select *HTTP>Listener*.
+. Drag *Listener* to the Studio canvas.
+. Set *Path* to `/account/{accountId}/logo`.
+. Click the plus sign (*+*) next to the *Connector configuration* field to configure a global element that can be used by all instances of HTTP *Listener* in the app.
+. Set *Host* to `0.0.0.0` and *Port* to `8081`.
+. Click *OK*.
+. In the HTTP *Listener* configuration screen, click the *Responses* tab.
+. In the *Response* section, set these values:
++
+* *Body* : `output text/plain --- vars.logoName ++ ' has been stored as a ' ++ vars.accountId ++ ' logo.'`
+* *Status code* : `201`
+* *Reason phrase* : `Created`
+
+[start=9]
+. In the *Error Response* section, set these values:
++
+* *Body* : `vars.logoName ++ ' could not be stored.'`
+* *Headers* :
++
+[source,DataWeave,linenums]
+----
+output application/java
+        ---
+        {
+          "X-Time" : "50s" // 4
+        }
+----
+* *Status code* : `vars.errorCode default 500`.
+
+image::http-listener-9.png[HTTP Listener Responses configuration window with Body, Header, Status Code and Reason phrase fields]
+
+[start=10]
+. Drag a *Set Variable* component to the right of HTTP *Listener*.
+. Set *Name* to `logoName`.
+. Set *Value* to `payload.parts.name.content`.
+. Drag another *Set Variable* component to the right of first *Set Variable* component.
+. Set *Name* to `accountId`.
+. Set *Value* to `attributes.uriParams.accountId`.
+. Drag an *Insert* operation to the right of the second *Set Variable* component.
+. Click the plus sign (*+*) next to the *Connector configuration* field to configure the database connection.
+. In the *Insert* operation configuration screen, set *SQL Query Text* to `INSERT INTO logo (accountId,logoName)`.
```

Comment:
Change the text to: `INSERT INTO logo (accountId,logoName) VALUES (:accountId, :logoName)`
Add this step:
. Set *Input Parameters* to `{'accountId': vars.accountId as String, 'logoName': vars.logoName as String}`